### ISSUE TITLE
Based Requests

### ISSUE THREAD

**Issue by Kardel420** [ColBeckett/Tank-accessory-pack#3]
I think the idea of adding premade tank parts is great, It can make most people save tons of props for little, dumb details.

So I have read the other 2 requests and I decided to bring up my own.

-Bodygrouped (If needed) and paintable parts
-Compatibility with submaterials (for maxium paint variety)
-Optical Parts (Probably Cold War: 1st 2nd and 3rd gen)
-Individual Periscopes
-Round Turret parts?
  -So much people likes round turrets, but they end up having so many props, for the protection they give, neither are effective or look good. I was thinking about round segments scaled to SProps Tubes and such for maxium scaleage. For example bent tubes and stretched bent tubes, streetch and bent probably like PHX parts, but better scaled, for stretched bent tubes i meant something like PHX's Rollercoaster tracks stretched curves.
-Baskets/Inventory
-Torsion bars
-More Tank wheels that aren't Sprops or Lanbox's?
-Mantlets

Not saying "DO NOW PLS" but The 1st 5 requests are the most I'd love to see in the future of this pack.

~Kardel

**Comment by ColBeckett** [comment]
this will add only tank details - not whole structure
so im afraid to say no to turret part :(
the only "armored" part will be mantlets cause there are rather small (compared to turret) and usually have really hard shapes that is hard todo with current props..

rounded turret can be easy done with sprops

**Comment by Kardel420** [comment]
Yes they can be done with sprops, except they dont have the same perfomance, especially with the health nerf, spamming tiny tubes is practically shit.

**Comment by Amplar** [comment]
I think suspension parts would be a good idea. Trunions, HVSS type stuff.

**Comment by ColBeckett** [comment]
i still think how todo that VVSS (HVSS is too complicated for now)
i could make it ragdol-like but there are 2cons of that:
- some servers have restricted ragdolls (you cant spawn any)
- i dont think is that possible to ball-socket part off it, even if its possible - that would be hard to ballsock wheel of suspension (propably cus ballsocket/axis center will take center of prop/ragdoll... i think)
  correct me if im wrong

2nd is to make that suspension as props (then you can make holo or axis that correctly)
cons:
- more props to spawn
- it may cause problems in heavy contraption

about torrison - i dont think is that a point doing that - there are some props already with can be usefull if you need this

**Comment by Amplar** [comment]
Only prop that's good for torsion arms is the Ep2 crank level, but it doesn't mirror well and most servers don't have ep2.
